lib/evss/letters/service.rb

```diff
@@ -66,8 +66,7 @@ def log_edipi
       end
 
       def invalid_address_error?(error)
-        return false unless error.is_a?(Common::Client::Errors::ClientError)
-        error&.body&.dig('messages')&.any? { |m| m['key'].include? INVALID_ADDRESS_ERROR }
+        error.body.dig('messages')&.any? { |m| m['key'].include? INVALID_ADDRESS_ERROR }
```

Comment:
if body is a string isn't dig still going to fail?

Reply:
Yea, but the call made to `invalid_address_error?` is within an error check that makes sure its a `Hash`